### PR TITLE
Change mha/mhd/nrrd to compound .igs extension

### PR DESCRIPTION
**Pull request by Sunderlandkyl**

### CODE REVIEW

**Comment by Sunderlandkyl** [comment]
@lassoan Updated the sequence file extensions here.
There will likely need to be some changes on the SlicerOpenIGTLink side as well.

**Comment by lassoan** [comment]
Thank you, it looks good. Add nhdr file extension, too (nrrd for nhdr is the same as mha for mhd).

**Comment by Sunderlandkyl** [comment]
nhdr added.

**Comment by adamrankin** [comment]
Out of curiosity, does .igs.mha imply a sequence of frames, or there is frame header data? Is a sequence of 1 frame a .igs.mha file or a .mha file?

**Comment by Sunderlandkyl** [comment]
From https://github.com/SlicerRt/Sequences/issues/61:

> .igs.nrrd and .igs.mha are new composite file extensions for sequence files. They allow Slicer (and users) to see that the file contains tracked image data, without looking into the file.

I would think that a sequence of 1 frame would be a .igs.mha if it had additional frame metadata. Regular image volumes from volume reconstruction, etc. are just .mha.

**Comment by adamrankin** [comment]
Ok, vtkPlusSequenceIO:: Write will have to be more intelligent (mha and need) or take arguments as my PR changes it to treat it as a sequence of 1 frame.

File: src/PlusCommon/PlusCommon.cxx [hunk 1]
```diff
@@ -316,6 +316,59 @@ PlusStatus PlusCommon::CreateTemporaryFilename(std::string& aString, const std::
...
+      ".nrrd",
+      ".nhdr",
+    }
+  );
+}
+
+//----------------------------------------------------------------------------
+std::string PlusCommon::GetSequenceFilenameWithoutExtension(std::string name)
+{
+  std::string extension = PlusCommon::GetSequenceFilenameExtension(name);
```
[the remark this answers is not in the document]
Comment (adamrankin) [reply src/PlusCommon/PlusCommon.cxx]: Ah, to support periods in file names. Gotcha.

**Comment by adamrankin** [comment]
Hmm, I think it will be ok. The requesting application will provide the filename, and it will know if the data is sequence or not.

**Pull request merged**